tests/test_main_task.py

```diff
@@ -51,6 +52,61 @@ def test_entity_results():
     assert [7] == entity_values(con, 1, 2)
 
 
+def test_timeseries():
+    reload(plugin_manager)
+    plugin_manager.init_plugin_manager()  # init plugin manager
+    plugin_manager.collect_plugins()
+
+    con = MagicMock()
+    con.lrange.return_value = [
+            '{"ts": 1480, "value": 1}',
+            '{"ts": 1450, "value": 1}',
+            '{"ts": 1420, "value": 2}',
+            '{"ts": 1390, "value": 1}',
+            '{"ts": 1360, "value": 1}',
+            '{"ts": 1330, "value": 1}',
+            '{"ts": 1300, "value": 1}',
+            '{"ts": 1270, "value": 1}',
+            '{"ts": 1240, "value": 1}',
+            '{"ts": 1210, "value": 1}',
+            '{"ts": 1180, "value": 1}',
+            '{"ts": 1150, "value": 1}',
+            '{"ts": 1120, "value": 1}',
+            '{"ts": 1090, "value": 1}',
+            '{"ts": 1060, "value": 1}',
+            '{"ts": 1030, "value": 1}',
+            '{"ts": 1000, "value": 1}',
+        ]
+    ts = build_condition_context(con, 1234, 2345, {'id': 'ent-1'}, {}, {})['timeseries_sum']
+    res = ts('5m')
+    assert con.lrange.called_once()
+    assert con.lrange.called_with('zmon:checks:1234:ent-1', 20)
```

Comment:
I guess the "20" should be  zmon_worker.tasks.main.DEFAULT_CHECK_RESULTS_HISTORY_LENGTH

Reply:
Agree. Also it would be less verbose if we switched the ``return_value`` to list comprehension instead:
``[{'ts': ..} for i in range(DEFAULT_CHECK_RESULTS_HISTORY_LENGTH)]``